python/ray/autoscaler/_private/constants.py

```diff
@@ -36,6 +36,3 @@ def env_integer(key, default):
 BOTO_MAX_RETRIES = env_integer("BOTO_MAX_RETRIES", 12)
 # Max number of retries to create an EC2 node (retry different subnet)
 BOTO_CREATE_MAX_RETRIES = env_integer("BOTO_CREATE_MAX_RETRIES", 5)
-
-# Host path that Docker mounts attach to
-DOCKER_MOUNT_PREFIX = "/tmp/ray_tmp_mount"
```

Comment:
@ijrsvt removing this will guarantee no one will use it accidentally. But it might break imports that currently use it in our other use cases.

Reply:
Good idea--I like that this makes it clear that we are deprecating the private API!